refinery/ui/source/js/file-browser/ctrls/ctrl.js

```diff
@@ -293,6 +324,18 @@
       }
     }
 
+    function isFiltered () {
+      var filtered = true;
+      if (paramService.fileParam.filter_attribute === undefined) {
+        filtered = false;
+      } else {
+        filtered = Boolean(
```

Comment:
? which use case is this resolving

Reply:
can we avoid this method all together?

Reply:
This is how I'm determining if the user has started using the facets to filter the results in the table. I need to have some notion of this happening to be able to either download a csv including all files in the dataset or just the filtered subset. I'm not sure if there is a better way to get that info